models/train_models.ipynb

```diff
@@ -27,7 +27,7 @@
   },
```

Comment:
<p>I am confused why the index looks 0 for all ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/github-labeler/pull/64/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
<p>Ah this is because I make a small dataframe at each iteration then concatenate them, should I fix this?</p>

Reply:
<p>Looks good, Thank you :) </p>